packages/time-utilities/src/lib/Timestamp.ts

```diff
@@ -0,0 +1,270 @@
+	['A', (time): string => (time.getHours() < 12 ? 'AM' : 'PM')],
+	['m', (time): string => String(time.getMinutes())],
+	['mm', (time): string => String(time.getMinutes()).padStart(2, '0')],
+	['s', (time): string => String(time.getSeconds())],
+	['ss', (time): string => String(time.getSeconds()).padStart(2, '0')],
+	['S', (time): string => String(time.getMilliseconds())],
+	['SS', (time): string => String(time.getMilliseconds()).padStart(2, '0')],
+	['SSS', (time): string => String(time.getMilliseconds()).padStart(3, '0')],
+	[
+		'T',
+		(time): string => `${String(time.getHours() % 12 || 12)}:${String(time.getMinutes()).padStart(2, '0')} ${time.getHours() < 12 ? 'AM' : 'PM'}`
+	],
+	[
+		't',
+		(time): string =>
+			`${String(time.getHours() % 12 || 12)}:${String(time.getMinutes()).padStart(2, '0')}:${String(time.getSeconds()).padStart(2, '0')} ${
+				time.getHours() < 12 ? 'am' : 'pm'
+			}`
+	],
+	[
+		'L',
+		(time): string => `${String(time.getMonth() + 1).padStart(2, '0')}/${String(time.getDate()).padStart(2, '0')}/${String(time.getFullYear())}`
+	],
+	['l', (time): string => `${String(time.getMonth() + 1)}/${String(time.getDate()).padStart(2, '0')}/${String(time.getFullYear())}`],
+	['LL', (time): string => `${months[time.getMonth()]} ${String(time.getDate()).padStart(2, '0')}, ${String(time.getFullYear())}`],
+	['ll', (time): string => `${months[time.getMonth()].slice(0, 3)} ${String(time.getDate()).padStart(2, '0')}, ${String(time.getFullYear())}`],
+	[
+		'LLL',
+		(time): string =>
+			`${months[time.getMonth()]} ${String(time.getDate()).padStart(2, '0')}, ${String(time.getFullYear())} ${String(
+				time.getHours() % 12 || 12
+			)}:${String(time.getMinutes()).padStart(2, '0')} ${time.getHours() < 12 ? 'AM' : 'PM'}`
+	],
+	[
+		'lll',
+		(time): string =>
+			`${months[time.getMonth()].slice(0, 3)} ${String(time.getDate()).padStart(2, '0')}, ${String(time.getFullYear())} ${String(
+				time.getHours() % 12 || 12
+			)}:${String(time.getMinutes()).padStart(2, '0')} ${time.getHours() < 12 ? 'AM' : 'PM'}`
+	],
+	[
+		'LLLL',
+		(time): string =>
+			`${days[time.getDay()]}, ${months[time.getMonth()]} ${String(time.getDate()).padStart(2, '0')}, ${String(time.getFullYear())} ${String(
+				time.getHours() % 12 || 12
+			)}:${String(time.getMinutes()).padStart(2, '0')} ${time.getHours() < 12 ? 'AM' : 'PM'}`
+	],
+	[
+		'llll',
+		(time): string =>
+			`${days[time.getDay()].slice(0, 3)} ${months[time.getMonth()].slice(0, 3)} ${String(time.getDate()).padStart(2, '0')}, ${String(
+				time.getFullYear()
+			)} ${String(time.getHours() % 12 || 12)}:${String(time.getMinutes()).padStart(2, '0')} ${time.getHours() < 12 ? 'AM' : 'PM'}`
+	],
+	[
+		'Z',
+		(time): string => {
+			const offset = time.getTimezoneOffset();
+			const unsigned = offset >= 0;
+			const absolute = Math.abs(offset);
+			/* istanbul ignore next: whether it's signed or unsigned, depends on where the machine is, I cannot control this. */
```

Comment:
Are these comments valid with our test suites?

Reply:
No.

Reply:
We're using Istanbul in here at least now. It could be changes to v8. I did also comment on this though, asking if why it cannot be reached in tests.